marketplace/deployer_util/wait_for_ready.py

```diff
@@ -178,8 +179,12 @@ def is_service_ready(resource):
 
 
 def is_ingress_ready(resource):
-  if 'ingress' in resource['status']['loadBalancer']:
-    return True
+  for key, value in annotations(resource).items():
+    if key == 'ingress.kubernetes.io/backends':
```

Comment:
I believe these annotations are gce-ingress specific; i.e. it only works for GKE. I don't know of a universal way to determine if an ingress is ready. My suggestions:
- If this is critical to the applications that we're testing, we can keep this logic and add a TODO(issue) to follow up.
- Otherwise, skip ingresses when checking for readiness.

Reply:
We have workaround for Ingress, see https://github.com/GoogleCloudPlatform/click-to-deploy/blob/master/k8s/wordpress/apptest/tester/tester.sh#L20-L30. However, having this integrated with the framework sounds better to me.

Do you think we should skip Ingress anyway?

Reply:
It's reasonable to do this. However, we should separate two cases:
- `ingress.kubernetes.io/backends` annotation exists
  - Return true iff at least one backend is `HEALTHY`
- `ingress.kubernetes.io/backends` annotation does not exist
    - We can fallback to the previous (imperfect) logic